.github/workflows/release-prep.yml

```diff
@@ -382,6 +381,11 @@ jobs:
         with:
           python-version: ${{ env.PYTHON_TARGET_VERSION }}
 
+      - name: "Install Spark Dependencies"
+        if: ${{ github.repository == 'dbt-labs/dbt-spark-release-test' }}
```

Comment:
Let's make this and env variable at the top to ensure it's always the same and make it easier to update when we move this process into the real repos.

Reply:
Fixed